templates/openwisp2/settings.py

```diff
@@ -173,6 +174,47 @@
 USE_L10N = True
 USE_TZ = True
 
+#Initialize django-cors-headers
+{% if cors:enable %}
```

Comment:
look at what we did in the gzip patch and do it in a similar way and add attributes only if present in the dictionary

Reply:
@nemesisdesign I looked at it and still have no idea what you are referring to :disappointed: . Please elaborate it.